sdk/python/kfp/components/_dynamic.py

```diff
@@ -44,21 +45,31 @@ def pass_locals():
         mod_co_filename = code.co_filename
         mod_co_firstlineno = code.co_firstlineno
 
-    modified_code = types.CodeType(
-        mod_co_argcount,
-        code.co_kwonlyargcount,
-        mod_co_nlocals,
-        code.co_stacksize,
-        code.co_flags,
-        code.co_code,
-        code.co_consts,
-        code.co_names,
-        mod_co_varnames,
-        mod_co_filename,
-        mod_co_name,
-        mod_co_firstlineno,
-        code.co_lnotab
-    )
+    if sys.version_info >= (3, 8):
+        modified_code = (code
+            .replace(co_argcount=mod_co_argcount)
```

Comment:
You don't need all these `replace` calls. You can simply do:

```
code.replace(co_argcount=mod_co_argcount, co_nlocals=mod_co_nlocals, co_varnames=mod_co_varnames, co_filename=mod_co_filename, co_name=mod_co_name, co_firstlineno=mod_co_firstlineno)
```

Reply:
Thanks!
It looks better now.